### PR TITLE
use react-components 20.0.0, update docs

### PR DESCRIPTION
**Pull request by davidjgoss**
### 🤔 What's changed?

- Use latest React components including with new package name
- Update documentation accordingly
- Add a readme to the javascript package that explains to users that it's internal (I have seen a few users in the wild installing it themselves)

### 🏷️ What kind of change is this?

- :bug: Bug fix (non-breaking change which fixes a defect)
- :zap: New feature (non-breaking change which adds new behaviour)

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)
- [ ] I've changed the behaviour of the code
  - [ ] I have added/updated tests to cover my changes.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] Users should know about my change
  - [x] I have added an entry to the "Unreleased" section of the [**CHANGELOG**](../CHANGELOG.md), linking to this pull request.

----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/PULL_REQUEST_TEMPLATE.md)*

### CODE REVIEW

**Review by aurelien-reeves — COMMENTED** [review]
Looks good

Just have to fix the CI

Also, could be nice to have the same notice in READMEs for ruby and java implementations

File: javascript/package.json [hunk 1]
```diff
@@ -76,5 +76,9 @@
   "directories": {
     "test": "test"
   },
-  "keywords": []
+  "keywords": [],
+  "dependencies": {
```
Comment (aurelien-reeves) [inline javascript/package.json:80]: Everything else seems to be declared as devDependencies. Does fortawesome package are fine as regular dependencies?

Comment (davidjgoss) [reply javascript/package.json:80]: Sorry - this was a quick experiment - going to make the depedency change in react-components and do a patch release there.

**Review by aurelien-reeves — APPROVED** [review]
👍

**Pull request merged**